docroot/modules/custom/sitenow_dcv/sitenow_dcv.links.menu.yml

```diff
@@ -1,5 +1,5 @@
 sitenow_dcv.settings:
-  title: 'SiteNow DCV'
+  title: 'DCV'
```

Comment:
I think this should be spelled out as Domain Control Validation.

Reply:
Should the description stay spelled out too, or do we want to switch to abbreviating that?

Reply:
I think its fine how it is now.